BCStudentSoftwareDevTeam/chemical-inventory
# Update python 3

## Did: 
updating the Python version from 2.7 to 3.7 and above and updating all the related libraries and syntax error 

## To Test:
- clone this Repo 
- make sure you have a Python version 3.7 or above installed. To check: run "python --version"
- Checkout the branch `update_python3`
-  run 'source setup.sh' and make sure nothing breaks
- reset the database run 'python reset-db.py'
- run 'python run.py" and check the application on your browser running.
- chick all the taps, add and delete chemicals and nothing breaks and see if they are being saved in the database
-  check any syntax errors from Python 2 such as print statements
